{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Product defects","children":[{"name":"Timed out retrying after 4000ms: Expected to find element: `:nth-child(1) > .product-item-info > .photo > .product-image-container > .product-image-wrapper > .product-image-photo`, but never found it.","children":[{"name":"Adding product to cart from home page","uid":"5037757af6fe18d1","parentUid":"aaf3c31e09f937a7f636a06cfff30f5e","status":"failed","time":{"start":1682539460879,"stop":1682539473014,"duration":12135},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]},{"name":"Visits the home page and logs in","uid":"97186ef23bebf550","parentUid":"aaf3c31e09f937a7f636a06cfff30f5e","status":"failed","time":{"start":1682539596513,"stop":1682539608793,"duration":12280},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"aaf3c31e09f937a7f636a06cfff30f5e"},{"name":"`cy.should()` failed because you invoked a command inside the callback. `cy.should()` retries the inner function, which would result in commands being added to the queue multiple times. Use `cy.then()` instead of `cy.should()`, or move any commands outside the callback function.\n\nThe command invoked was:\n\n  > `cy.checkCartNotEmpty()`","children":[{"name":"Adding product to cart from a category","uid":"c243ae6d0128f68a","parentUid":"76514becae13a6566894a84a107d8c79","status":"failed","time":{"start":1682539473036,"stop":1682539497403,"duration":24367},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"76514becae13a6566894a84a107d8c79"},{"name":"The following error originated from your application code, not from Cypress.\n\n  > $(...).AddFotoramaVideoEvents is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","children":[{"name":"Adding product to cart from search","uid":"98f6b2be8e319f9","parentUid":"3899f489dc56f2ea95ec8c11525e9020","status":"failed","time":{"start":1682539497437,"stop":1682539512056,"duration":14619},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"3899f489dc56f2ea95ec8c11525e9020"},{"name":"Timed out retrying after 5000ms: Expected to find content: 'A login and a password are required' but never did.","children":[{"name":"Login testing required fields","uid":"d4e131c0b0e1a61d","parentUid":"ce7fccfafd744aa5dd7f8181472fd537","status":"failed","time":{"start":1682539564444,"stop":1682539583159,"duration":18715},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"ce7fccfafd744aa5dd7f8181472fd537"}],"uid":"8fb3a91ba5aaf9de24cc8a92edc82b5d"}],"name":"categories"}